addons/web/static/src/js/core/abstract_service.js

```diff
@@ -7,14 +7,37 @@ var ServicesMixin = require('web.ServicesMixin');
 
 var AbstractService = Class.extend(Mixins.EventDispatcherMixin, ServicesMixin, {
     dependencies: [],
-    init: function (parent) {
+    init: function (env) {
```

Comment:
should use short syntax

Reply:
I disagree: it would be inconsistent with the rest of this file, and would kind of drown the purpose of this commit into a larger diff. We'll rewrite this in the future.

Reply:
Sounds good